src/audio/codec_adapter/codec/cadence.c

```diff
@@ -172,6 +172,47 @@ int cadence_codec_init(struct comp_dev *dev)
 	return ret;
 }
 
+static int update_stream_params(struct comp_dev *dev)
+{
+	int ret = 0;
```

Comment:
IIRC `API_CALL()` was `ret = ...` in which case this initialisation is superfluous